libc/src/__support/FPUtil/x86_64/LongDoubleBits.h

```diff
@@ -33,37 +33,37 @@ template <> struct FPBits<long double> : private FloatProperties<long double> {
   using FloatProperties<long double>::EXPONENT_MASK;
   using FloatProperties<long double>::EXPONENT_BIAS;
   using FloatProperties<long double>::EXPONENT_WIDTH;
-  using FloatProperties<long double>::MANTISSA_MASK;
-  using FloatProperties<long double>::MANTISSA_WIDTH;
+  using FloatProperties<long double>::FRACTION_MASK;
+  using FloatProperties<long double>::FRACTION_BITS;
```

Comment:
My only concern with this terminology is the inconsistency of : `BIT_WIDTH`, `EXPONENT_WIDTH` vs `FRACTION_BITS`

Reply:
I'm willing to change them to:
 - `EXPONENT_WIDTH` => `EXP_BITS` - or `EXPONENT_BITS` if you prefer
 - `BIT_WIDTH` => `TOTAL_BITS` - or any other suggestions, I initially thought about `FP_BITS` but @michaelrj-google rightfully suggested that it would conflict with the `FPBits` class.

I can do the modification as part of this patch or in another one if you prefer, let me know.

Reply:
I was thinking that using `LENGTH` instead of `WIDTH` would make it clearer:  `BIT_LENGTH` or `TOTAL_LENGTH`, `EXPONENT_LENGTH`, `FRACTION_LENGTH`.  WDYT?

Reply:
I like it.
How about `_LEN` to make them shorter? I'm fine with `_LENGTH` also.

Reply:
Hmm but then it doesn't quite work with `UINTTYPE_BITS` -> `UINTTYPE_LENGTH` as it's unclear whether we count bits or bytes.

Reply:
`_LEN` SGTM.
One way of resolving the `UINTTYPE_BITS` that I was thinking is to change `UIntType` to `StorageType` and then `STORAGE_LEN` would make sense.  WDYT?

Reply:
I'll give it a try so we can evaluate whether it looks OK or not.